tools/make_project

```diff
@@ -326,6 +326,12 @@ except:
 
 print '''Done installing default daemons.'''
 
+try:
+    os.system('cd '+proot+'/html/ops; ./generate_account_ownership_keys.php')
+    print '''Generated account ownership keypair'''
```

Comment:
Make sure that the call succeeds before saying that it was generated successfully.  I ran the make project script and it failed to generate the files but reported success.  (see https://docs.python.org/2/library/os.html#os.system)

Reply:
Did it fail to generate because it didn't have the executable flag applied?

Reply:
Yes